srp_daemon/srp_daemon.c

```diff
@@ -1619,7 +1644,8 @@ static int get_config(struct config_t *conf, int argc, char *argv[])
 	while (1) {
 		int c;
 
-		c = getopt(argc, argv, "caveod:i:j:p:t:r:R:T:l:Vhnf:");
+		optind = 1;
```

Comment:
Yep, I'm going to move these two patches to another series and post them to the list. This has gone beyond packaging changes